src/main/java/com/revature/ams/Booking/BookingService.java

```diff
@@ -45,7 +45,9 @@ public List<Booking> findAll() {
     }
 
     /**
-     * TODO: DOCUMENT ME
+     * This service method retrieves all bookings made by a specific member based on their memberId.
+     * The sends data generated from the bookingRepository and sends it to the bookingController.
```

Comment:
Small typo should be Then rather then The

Reply:
![noooo-emote](https://github.com/user-attachments/assets/a310bfdf-e716-429f-b92d-c33be0e31d5a)